.github/workflows/views-core.yml

```diff
@@ -0,0 +1,32 @@
+name: Core Views Tests
+on:
+  pull_request:
+    paths:
+      # Direct view changes
+      - 'src/Tribe/Views/V2/Views/**'
```

Comment:
The whole `src/Events` directory is missing from the paths. A number of files there will have a impact on the the Views, I think.